### DISCUSSION TITLE
Deployment's build step did not complete within the maximum of 45min

### DISCUSSION THREAD

**Discussion by joinpoc** [vercel/community#4529] [Help] [answered]
### Summary

My Build failed with error 'Build Failed
Your deployment's build step did not complete within the maximum of 45min'

How will Vercel help here to increase the time limit of build.  

Wish I do get helping hands here. Thanks

### Steps to Reproduce

App link is - [strapipoc-b0c5skxdi-joinpocs-projects.vercel.app](https://vercel.live/link/strapipoc-b0c5skxdi-joinpocs-projects.vercel.app?via=deployment-domains-list-commit)

**Comment by amyegan** [comment, marked as answer]
The maximum [build time per deployment](https://vercel.com/docs/limits/overview#build-time-per-deployment) is 45 minutes for all plans. I can't increase that limit for you, but there are strategies, such as [Incremental Static Regeneration (ISR)](https://vercel.com/docs/incremental-static-regeneration#), that can help reduce build time.

https://vercel.com/docs/deployments/troubleshoot-a-build#troubleshoot-build-time
